src/languages/rust.ts

```diff
@@ -146,6 +146,7 @@ const nodeMatchers: Partial<
   ),
   string: ["raw_string_literal", "string_literal"],
   ifStatement: ["if_expression", "if_let_expression"],
+  condition: leadingMatcher(["*.match_pattern![condition]"], ["if"]),
```

Comment:
I'm surprised this wasn't implemented in the first place. We should make sure this handles `if` and `while` conditions too.

Open question for how to handle `if let` and `while let`, though I imagine it should take the whole `let <match_pattern> = <value>`.

Reply:
Done